### PR TITLE
angle-grinder: update to 0.12.0

### PR DESCRIPTION
**Pull request by herbygillot**
#### Description

Upgrade `angle-grinder` to version 0.12.0
The `logfmt` package is now a standard crate, so we no longer need the patches and hacks needed to bring it in through MacPorts as a standalone Github-hosted crate.

###### Tested on

macOS 10.15 19A583
Xcode 11.1 11A1027

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -d install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: sysutils/angle-grinder/Portfile [hunk 1]
```diff
@@ -20,13 +20,12 @@ long_description    Angle-grinder allows you to parse, aggregate, sum, \
                     your data in graphite/honeycomb/kibana/sumologic/splunk/etc. \
                     but still want to be able to do sophisticated analytics.
 
-depends_lib-append  path:lib/libssl.dylib:openssl
+checksums-append    ${name}-${version}${extract.suffix} \
```
Comment (mf2k) [inline sysutils/angle-grinder/Portfile]: `${name}-${version}${extract.suffix}` can more simply be specified using `${distfiles}`.

Comment (herbygillot) [reply sysutils/angle-grinder/Portfile]: Would `${distfiles}` not be affected by all the crates?

**Comment by cjones051073** [comment]
This now doesn’t update the version, just removes patch files ?

**Comment by herbygillot** [comment]
> This now doesn’t update the version, just removes patch files ?

It does both; version bump and patchfile removal (you have to load the Portfile diff to see, for some reason the Portfile diff is redacted even though it's not all deletions)

**Comment by cjones051073** [comment]
Sorry, so it does, my mistake. Must have not read the small print under the portfile changes correctly...

**Pull request merged**